### PR TITLE
EVG-5923: initial jasper integration

### PR DESCRIPTION
**Pull request by tychoish**

### CODE REVIEW

File: agent/agent.go [hunk 1]
```diff
@@ -67,28 +69,37 @@ type taskContext struct {
...
 	comm.SetHostID(opts.HostID)
 	comm.SetHostSecret(opts.HostSecret)
 	agent := &Agent{
 		opts: opts,
 		comm: comm,
 	}
 
-	return agent
+	jpm, err := jasper.NewLocalManager(true)
+	if err != nil {
```
Comment (john-m-liu) [inline agent/agent.go]: Are you sure you want errors creating a process manager to prevent starting the agent?

File: util/json.go [hunk 2]
```diff
@@ -22,6 +23,7 @@ func ReadJSONIntoWithLength(r io.ReadCloser, data interface{}) (int, error) {
 		return 0, errors.Wrapf(err, "error reading JSON (%s)", string(bytes))
 	}
 	length := len(bytes)
+	fmt.Println("-->", string(bytes))
```
Comment (john-m-liu) [inline util/json.go]: left some debug code

File: makefile [hunk 3]
```diff
@@ -250,8 +250,8 @@ vendor-clean:
 	rm -rf vendor/github.com/mongodb/amboy/vendor/github.com/aws/
 	rm -rf vendor/github.com/mongodb/amboy/vendor/github.com/mongodb/mongo-go-driver/vendor/golang.org/x/net/
 	rm -rf vendor/github.com/mongodb/amboy/vendor/github.com/mongodb/mongo-go-driver/vendor/golang.org/x/text/
-	rm -rf vendor/github.com/mongodb/jasper/vender/github.com/mongodb/grip
-	rm -rf vendor/github.com/mongodb/jasper/vender/github.com/mongodb/amboy
+	rm -rf vendor/github.com/mongodb/jasper/vendor/github.com/mongodb/grip/
+	rm -rf vendor/github.com/mongodb/jasper/vender/github.com/mongodb/amboy/
```
Comment (john-m-liu) [inline makefile]: These 3 are also misspelled

File: agent/agent.go (hunk 1 above)
Comment (tychoish) [reply agent/agent.go]: I think it's unavoidable.

**Pull request merged**